service.php

```diff
@@ -63,21 +63,27 @@ function page_optimize_service_request() {
 		}
 	}
 
-	$output = page_optimize_build_output();
+	list(
```

Comment:
Please no list... Explicit assignment (tho this is my preference, all of these magic functions that have the same # of rows as regular assignment are not worth it. Destructuring in PHP isn't preferred (and is harder to read, imho).

Reply:
I wondered if you'd say that. 😄  Personally, I prefer not creating a separate variable simple to assign other variables from, and I like reading a single statement that shows what we're getting from the function call. That said, my view is the minority view in my experience, and if it isn't idiomatic PHP or preferred by our team, I'm happy to avoid it.

Reply:
Oops... didn't realize you changed it already. I made the change, rebased and had a conflict.

thanks!